lib/ain-evm/src/executor.rs

```diff
@@ -187,7 +188,10 @@ impl<'backend> AinExecutor<'backend> {
                 signed_tx.sender,
                 signed_tx.gas_limit(),
                 U256::from(used_gas),
-                signed_tx.gas_price(),
+                match base_fee {
+                    Some(base_fee) => signed_tx.effective_gas_price(base_fee),
```

Comment:
We should be refactoring refund_unused_gas total gas fee calculation to use the 'calculate_gas_fee' helper function in fee lib.